src/components/v1/V1Project/ProjectActivity/index.tsx

```diff
@@ -7,321 +7,120 @@ import DeployedERC20EventElem from 'components/activityEventElems/DeployedERC20E
-        keys: ['id', 'txHash', 'timestamp', 'from'],
-      },
-      {
-        entity: 'v1ConfigureEvent',
-        keys: [
-          'id',
-          'timestamp',
-          'txHash',
-          'from',
-          'ballot',
-          'discountRate',
-          'duration',
-          'target',
-          'bondingCurveRate',
-          'reservedRate',
-          'currency',
-          'ticketPrintingIsAllowed',
-          'payIsPaused',
-        ],
-      },
-    ],
-    orderDirection: 'desc',
-    orderBy: 'timestamp',
-    where,
+  const { data, fetchMore, loading } = useProjectEvents({
+    pv: PV_V1,
+    projectId,
+    filter: eventFilter,
+    first: PAGE_SIZE,
   })
 
-  const count =
-    projectEvents?.pages?.reduce((prev, cur) => prev + cur.length, 0) ?? 0
+  const projectEvents = data?.projectEvents
+
+  const count = projectEvents?.length || 0
 
   const list = useMemo(
     () =>
-      projectEvents?.pages.map(group =>
-        group.map(e => {
-          let elem: JSX.Element | undefined = undefined
-
-          if (e.payEvent) {
-            elem = <PayEventElem event={e.payEvent as PayEvent} />
-          }
-          if (e.burnEvent) {
-            elem = (
-              <BurnEventElem event={e.burnEvent} tokenSymbol={tokenSymbol} />
-            )
-          }
-          if (e.addToBalanceEvent) {
-            elem = (
-              <AddToBalanceEventElem
-                event={e.addToBalanceEvent as AddToBalanceEvent}
-              />
-            )
-          }
-          if (e.tapEvent) {
-            elem = <TapEventElem event={e.tapEvent as TapEvent} />
-          }
-          if (e.redeemEvent) {
-            elem = <RedeemEventElem event={e.redeemEvent as RedeemEvent} />
-          }
-          if (e.printReservesEvent) {
-            elem = (
-              <ReservesEventElem
-                event={e.printReservesEvent as PrintReservesEvent}
-              />
-            )
-          }
-          if (e.projectCreateEvent) {
-            elem = (
-              <ProjectCreateEventElem
-                event={e.projectCreateEvent as ProjectCreateEvent}
-              />
-            )
-          }
-          if (e.deployedERC20Event) {
-            elem = (
-              <DeployedERC20EventElem
-                event={e.deployedERC20Event as DeployedERC20Event}
-              />
-            )
-          }
-          if (e.v1ConfigureEvent) {
-            elem = (
-              <V1ConfigureEventElem
-                event={e.v1ConfigureEvent as V1ConfigureEvent}
-              />
-            )
-          }
-
-          if (!elem) return null
-
-          return (
-            <div
-              className="mb-5 border-b border-smoke-200 pb-5 dark:border-grey-600"
-              key={e.id}
-            >
-              {elem}
-            </div>
-          )
-        }),
-      ),
+      projectEvents?.map(e => {
+        let elem: JSX.Element | undefined = undefined
+
+        if (e.payEvent) {
```

Comment:
this rendering function as well as the one used in the v2v3 dashboard could be replaced with a single PV-agnostic function. leaving out of the scope of this PR